modules/tools/pom.xml

```diff
@@ -130,6 +130,14 @@
                     <skip>false</skip>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
This is the reason you are getting Travis errors. Please roll this change back. Probably done by your IDE under the hood.